package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@driftory/viewer",
-  "version": "0.1.7",
+  "version": "0.1.8",
```

Comment:
Since a new API feature is being introduced that would suggest the version number should be 0.2.0 now according to semantic versioning. Don't worry about changing the version now though.

Sam-ver is:
BreakingChange.NewFeature.BugFix

Though while still in 0.x.x breaking changes are allowed until the official 1.x.x release is published.

Reply:
Good reminder. At this point, I think pretty much everything is a new feature, since the project is so fresh. Soon we'll be up to version 0.128.0 ;-)